notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # Package versions we used:scikit-learn==1.2.0 sentence-transformers==2.2.2
</pre><p>If you need these exact package versions (and not the newest), you can fix the versions in the <code>pip install</code> part, and remove this comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>